### PR TITLE
ENH/BLD: no cogent dependency, support for sortmerna

### PR DESCRIPTION
**Pull request by wasade**
~~DO NOT MERGE~~

~~Missing tests, need to update licenses.~~

Good for review and merge.

### CODE REVIEW

**Comment by antgonza** [comment]
The code looks fine but as you said, you are missing tests for parse_m9, update licenses & tutorial, and you have flake8 errors. However, really close.

**Comment by ElDeveloper** [comment]
Also update setup.py to remove cogent, update the license in setup.py,
and update the headers in the files to change GPL -> BSD (woot!!)

**Comment by wasade** [comment]
@antgonza @ElDeveloper, just need to do the sortmerna test. ran out of time this morning before needing to head out, should only take another 15 though so assuming wifi works on my laptop, i'll have it in before noon

**Comment by antgonza** [comment]
Thanks for the update.

**Comment by wasade** [comment]
@antgonza @ElDeveloper, okay, should be good for review.

File: COPYING.txt [hunk 1]
```diff
@@ -1,674 +1,43 @@
...
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of the BIOM Format Development Team nor the names of its
```
Comment (ElDeveloper) [inline COPYING.txt]: :%s/BIOM Format/Platypus Conquistador/gc

File: platypus/commands.py [hunk 2]
```diff
@@ -209,7 +209,7 @@ def split_db(tax_fp, seqs_fp, query, output_fp, split_fp):
     interest_fp = open(join(output_fp, 'interest.fna'), 'w')
     rest_fp = open(join(output_fp, 'rest.fna'), 'w')
 
-    for full_name, seq in MinimalFastaParser(open(seqs_fp, 'U')):
+    for full_name, seq in parse_fasta(seqs_fp):
```
Comment (ElDeveloper) [inline platypus/commands.py]: Since parse_fasta is already tagged for deprecation, should we use read instead?

Comment (antgonza) [inline platypus/commands.py]: Note that you are missing tests for those cases where you pass a
string vs. a file. You can see the missing lines/tests here:
https://coveralls.io/builds/2408540/source?filename=platypus%2Fparse.py

File: platypus/parse.py [hunk 3]
```diff
@@ -1,13 +1,96 @@
...
+    lines : iterable of str, or a filepath
+        The lines to parse. It is expected that these lines are in BLAST m9
+        format, or comparable. Any additional columns will be ignored
+
+    Returns
+    -------
+    list of namedtuple
+        The namedtuples describe each field of the m9 format per record.
+    """
+    if isinstance(lines_or_fp, str):
```
Comment (ElDeveloper) [inline platypus/parse.py]: Would you mind using skbio.io.util.open_file?

File: platypus/parse.py [hunk 4]
```diff
@@ -1,13 +1,96 @@
...
+                res.append((None, copy(M9_empty)))
+
+            continue
+
+        # BLAST output contains 12 fields, SortMeRNA has 14. The order is the
+        # same, and we only care about the 12 fields common with BLAST.
+        parts = line.strip().split('\t')[:12]
+
+        if len(parts) < 12:
+            raise ValueError("Unexpected number of fields found")
```
Comment (ElDeveloper) [inline platypus/parse.py]: Any way we can have a test for this exception?

**Comment by ElDeveloper** [comment]
Thanks for updating this! Would you mind also updating INSTALL.md to remove pycogent from that list?

File: platypus/parse.py (hunk 3 above)
Comment (wasade) [inline platypus/parse.py]: Was only being used via file object anyway so removed support for passing a
file path as thee isn't a need

File: platypus/parse.py (hunk 4 above)
Comment (wasade) [inline platypus/parse.py]: done

**Comment by wasade** [comment]
I think the comments are addressed.

File: platypus/parse.py (hunk 3 above)
Comment (ElDeveloper) [inline platypus/parse.py]: :boom:

**Comment by antgonza** [comment]
can you add a test that raises the valueerror due to having more than
12 fields in an input file?

**Comment by wasade** [comment]
`ValueError` is now specific

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**